docs/source/daml2ts/index.rst

```diff
@@ -0,0 +1,290 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+TypeScript!
+###########
+
+daml2ts : TypeScript of DAML
+============================
+
+Introduction
+------------
+
+``daml2ts`` generates TypeScript for interacting with DAML via the `json-api <../json-api/index.html>`_ .
+
+``daml2ts`` inputs are compiled DAML modules. ``daml2ts`` outputs are TypeScript source files containing definitions modeling the DAML entities found.
+
+The code ``daml2ts`` generates is in terms of the package `@daml/types <https://github.com/digital-asset/daml/tree/master/language-support/ts/daml-types>`_.
+
+Primitives: @daml/types
+-----------------------
+
+To understand the TypeScript code generated by ``daml2ts``, it is helpful to keep in mind this quick review of the primitives provided by @daml/types.
+
+**Interfaces**:
+
+- ``Serializable<T>``
```

Comment:
Let's not tell our users about this one. It's an implementation detail and doesn't make too much sense when you don't also talk about the companion objects, which I would like to avoid.